src/modules/Instabug.ts

```diff
@@ -247,25 +238,23 @@ function handleAndroidNativeInterception() {
 }
 
 /**
- * Control either to enable or disable the native interception logic for iOS.
+ * Control either to enable or disable the native interception for iOS after the init method is called.
  */
 function handleIOSNativeInterception(config: InstabugConfig) {
-  if (Platform.OS === 'ios') {
-    console.log(
-      `Andrew: handleIOSNativeInterception(${
-        shouldEnableNativeInterception &&
-        config.networkInterceptionMode === NetworkInterceptionMode.native
-      })`,
-    );
-
-    if (
+  console.log(
+    `Andrew: handleIOSNativeInterception(${
```

Comment:
I think this log should be removed.

Reply:
There is a **TODO** for removing logs. 👍